Library/Homebrew/cmd/link.rb

```diff
@@ -1,3 +1,17 @@
+#:  * `ln`, `link` [`--overwrite`] [`--dry-run`] [`--force`] <formula>:
```

Comment:
I think it would be a good idea to make the canonical name (`link`) the first and list the aliases (`ln`) afterwards. Otherwise the order in the generated man page will be weird sometimes.